### DISCUSSION TITLE
Code review IM-11/13

### DISCUSSION THREAD

**Discussion by AidXylelele** [HowProgrammingWorks/LiveQA#111] [Code review]
Мы занимаемся разработкой web-сервиса для быстрого создания блок-схем кода, написанного на языке C.
**Основа проекта - парсер кода С(готов).**

- Визуализация - Canvas(готова на 80%).
- Преобразование текста на RegExp (готово на 80%).

_В проекте не используются сторонние библиотеки и фреймворки._
Используется метод Fetch для реализации отображения аватарки пользователя и ссылки на его Github аккаунт, совершившего последний commit в наш репозиторий!
**[Наш репозиторий](https://github.com/chyrkovmaksym/course_project)
[Guthub pages](https://chyrkovmaksym.github.io/course_project/index.html)**

@tshemsedinov

**Comment by tshemsedinov** [comment]
See review: https://github.com/chyrkovmaksym/course_project/issues/2
